app/src/main/java/corp/kairos/adamastor/UserContext.java

```diff
@@ -15,17 +21,49 @@
 public class UserContext {
     private String contextName; //The name of the context
     private List<AppDetail> contextApps; //The list of apps associated with this context
+    private Location location;
+    private GregorianCalendar init;
```

Comment:
We're using the GregorianCalendar for begin and end hour, is that right?

Reply:
Yes